core/runtime/ops.rs

```diff
@@ -1865,6 +1867,20 @@ mod tests {
     resource.value
   }
 
+  #[op2(async)]
+  #[cppgc]
+  pub async fn op_test_make_cppgc_resource_async() -> TestResource {
+    TestResource { value: 42 }
+  }
+
+  #[op2(async)]
+  #[smi]
+  pub async fn op_test_get_cppgc_resource_async(
+    #[cppgc] resource: &TestResource,
+  ) -> u32 {
+    resource.value
+  }
```

Comment:
Can you do these in `checkin` instead. We don't want them added for all programs

Reply:
this is in `mod test` i.e. test runtime

Reply:
I think it's probably best to do as a checkin test tbh.